keyboards/handwired/hacked_motospeed/keymaps/german/keymap.c

```diff
@@ -0,0 +1,10 @@
+#include QMK_KEYBOARD_H
+
```

Comment:
Actually, you may want to add this: 
```suggestion
#include "keymap_german.h"

```

And then you could use these: 
https://github.com/qmk/qmk_firmware/blob/master/quantum/keymap_extras/keymap_german.h

Reply:
I did the whole layout through the qmk web configurator. And I'm pretty happy with how it turned out, I wouldn't really want to reimplement all my custom layered fuckery, which I needed to for example place all the open brackets ([<{ on one key and all the closed ones on another.

Reply:
Okay, not a problem. 

I mostly just wanted to make sure that you were aware of this file, and that it can be used.  

Though, it may be worth considering in the future, though that's completely up to you.